tests/resources/Group23-VIC-Machine-Service-Util.robot

```diff
@@ -150,3 +150,11 @@ Verify VCH List Empty
     ${vchs}=  Run  echo '${OUTPUT}' | jq -r '.vchs[]'
     Log  ${vchs}
     Length Should Be  ${vchs}  0
+
+Get Docker Params API
+    [Arguments]    ${vch}    ${vch_name}
+    Get Path Under Target    vch
```

Comment:
${vch} ?  otherwise that param is not used at all?

Reply:
Nice catch, going to fix that.  It works like this since there is only one "vch" in play when I use this.  But this may not be the case for future tests.